src/App.tsx

```diff
@@ -8,39 +8,36 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
-import { getUserPosts } from './api/posts';
 import { User } from './types/User';
-import { Post } from './types/Post';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { init } from './features/Posts';
+import { clear } from './features/Users';
+import { setSelectedPost } from './features/SelectedPost';
 
 export const App: React.FC = () => {
-  const [posts, setPosts] = useState<Post[]>([]);
-  const [loaded, setLoaded] = useState(false);
-  const [hasError, setError] = useState(false);
-
   const [author, setAuthor] = useState<User | null>(null);
```

Comment:
Not fixed:

![image](https://github.com/mate-academy/react_redux-list-of-posts/assets/86923089/b942839f-4259-4bcf-a18c-ee130209027f)

Required, check task description:
![image](https://github.com/mate-academy/react_redux-list-of-posts/assets/86923089/385601ba-9b16-4c0b-935f-af509ff58866)